docs/guides/react_native.md

```diff
@@ -80,6 +79,10 @@ npx pod-install
 
 Close the react native packager, clean the project, and rerun your app. If everything is okay, you should see the default React Native screen as before.
```

Comment:
If we just remove this line we could drop the troubleshooting text altogether, since`events` will be installed by `veramo/core` in the next step.

Reply:
Sounds good.  I'll test it out and do that as soon as I can.

Reply:
OK, I've changed that.  (I tested with iOS and Android successfully to that point in the instructions.)